phalcon/config.zep

```diff
@@ -223,6 +270,29 @@ class Config implements \ArrayAccess, \Countable
 	}
 
 	/**
+	 * Sets the default path delimiter
+	 */
+	public static function setPathDelimiter(string! delimiter = null) -> void
```

Comment:
But he wrote what is purposed of this, to configure delimiter once for all Config instances.